packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts

```diff
@@ -0,0 +1,114 @@
+import { css } from "@microsoft/fast-element";
+import {
+    display,
+    focusVisible,
+    forcedColorsStylesheetBehavior,
+} from "@microsoft/fast-foundation";
+import {
+    accentForegroundActiveBehavior,
+    accentForegroundHoverBehavior,
+    accentForegroundRestBehavior,
+    neutralForegroundRestBehavior,
+    heightNumber,
+} from "../styles/index";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+
+export const BreadcrumbItemStyles = css`
+    ${display("inline-flex")} :host {
+        background: transparent;
+        box-sizing: border-box;
+        font-family: var(--body-font);
+        font-size: var(--type-ramp-base-font-size);
+        fill: currentColor;
+        line-height: var(--type-ramp-base-line-height);
+        min-width: calc(${heightNumber} * 1px);
+        outline: none;
+    }
+
+    .listitem {
+        display: flex;
+        align-items: center;
+    }
+
+    .separator {
+        margin: 0 12px;
+    }
+
+    .control {
+        background: transparent;
+        box-sizing: border-box;
+        color: ${accentForegroundRestBehavior.var};
+        cursor: pointer;
+        fill: inherit;
+        outline: none;
+        text-decoration: none;
+        white-space: nowrap;
+    }
+
+    .control:hover {
+        color: ${accentForegroundHoverBehavior.var};
+    }
+
+    .control:active {
+        color: ${accentForegroundActiveBehavior.var};
+    }
```

Comment:
I'm noticing a couple of things:

1. Anchors without an href are being styled as anchors - we aren't going to want that. That will be an issue overall.
2. We likely want the item with `aria-current` to be styled differently than the other items. 
> The aria-current attribute is used when an element within a set of related elements is visually styled to indicate it is the current item in the set. For example:
>
>A page token used to indicate a link within a set of pagination links, where the link is visually styled to represent the currently-displayed page.

It's possible that the styles for no href and aria-current are similar; though I would expect no href to not have hover styles (it should be treated as text), and the aria-current would include hover styles matching it's visual style.

Reply:
I will check with design on this, but here is an update where the last item has no href and no aria-current. It is style to be a text.
![image](https://user-images.githubusercontent.com/37851220/89807201-1f923000-daed-11ea-9ef7-c1c2a1678d5a.png)

Reply:
worked with @bheston on design.